bitsail-connectors/connector-ftp/src/main/java/com/bytedance/bitsail/connector/ftp/source/reader/deserializer/JsonRowDeserializer.java

```diff
@@ -0,0 +1,163 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package com.bytedance.bitsail.connector.ftp.source.reader.deserializer;
+
+import com.bytedance.bitsail.common.BitSailException;
+import com.bytedance.bitsail.common.configuration.BitSailConfiguration;
+import com.bytedance.bitsail.common.model.ColumnInfo;
+import com.bytedance.bitsail.common.row.Row;
+import com.bytedance.bitsail.common.util.FastJsonUtil;
+import com.bytedance.bitsail.connector.ftp.error.FtpErrorCode;
+import com.bytedance.bitsail.connector.ftp.option.FtpReaderOptions;
+
+import com.alibaba.fastjson.JSONArray;
+import com.alibaba.fastjson.JSONObject;
+import com.alibaba.fastjson.serializer.SerializerFeature;
+
+import java.io.IOException;
+import java.util.List;
+import java.util.function.Function;
+import java.util.stream.Collectors;
+
+public class JsonRowDeserializer implements ITextRowDeserializer {
```

Comment:
hi @hk-lrzy ,I used **com.bytedance.bitsail.component.format.json.JsonDeserializationSchema**  instead of **JsonRowDeserializer**,  Added **case-sensitivity** and **convertErrorColumnAsNull** in **JsonDeserializationSchema**.

created a new module for the CSV which named **bitsail-component-format-csv**.

please review again!